src/components/main/files/upload/mod.rs

```diff
@@ -334,7 +334,7 @@ async fn set_thumbnail_if_file_is_image(
 
     if !file.is_empty() || !mime.is_empty() {
         let prefix = format!("data:{};base64,", mime);
-        let base64_image = base64::encode(image_thumbnail.as_bytes());
+        let base64_image = base64::encode(&file);
```

Comment:
For some reason it breaks the img component src link, I legit have no idea why

Reply:
Ill take a closer look and see here shortly because doing this pretty much reverts a different pr to using the image itself rather than passing through the library itself to resize it for a thumbnail (which base on the code is what its trying to do).

Reply:
sounds good

Reply:
Ran into an issue on my end with the dialog not loading up on uplink so im unable to test directly so i will go on and allow this for now although i think the correct thing for near term is to figure out why its causing a problem.